### ISSUE TITLE
Associations imported as null value - PMID 32929287

### ISSUE THREAD

**Issue by earlEBI** [EBISPOT/goci#615]
For PMID '32929287', original author submission, 731 studies published, then unpublished via the backend to add associations. 
I reuploaded the submission template via the depocuration app with associations and received no metadata errors, only sumstats errors, as expected:

![Screenshot 2022-02-14 at 08.47.20.png](https://images.zenhubusercontent.com/5e95cce7cc796e84ca614069/c13369b2-7c13-47b1-8e6d-f6ee0524392b)

This was pushed through by Sajo and imported by me. 

On checking, I see the most significant snps have imported with null p-values: 

![Screenshot 2022-02-14 at 08.15.58.png](https://images.zenhubusercontent.com/5e95cce7cc796e84ca614069/9a3d1441-6195-4396-aabe-60d66e58d64e)

I suspect all p-values below about 1e-308 are affected. See submission template: 

[https://app.zenhub.com/files/2995118/e7ba0f14-9955-4394-b352-074f823adda6/download](https://app.zenhub.com/files/2995118/e7ba0f14-9955-4394-b352-074f823adda6/download)

For this publication, I will fix the p-values manually to allow DR to go ahead. But the issue should be looked into to avoid the same issue reoccuring.

**Comment by ljwh2** [comment]
@earlEBI do you know if you would have been able to publish with this error? Looks like the associations are approved?

**Comment by earlEBI** [comment]
@ljwh2  Yes, they approved, but I'm afraid I don't know if it would've let me publish. I didn't try.
